### DISCUSSION TITLE
XML压缩器 - Bug Report

### DISCUSSION THREAD

**Discussion by MrAccle** [he3-app/he3-feedback#244] [Bug Report]
xml的压缩后丢失关闭标签

压缩前：
![image](https://github.com/he3-app/he3-feedback/assets/25242380/3ec5c2ec-ec26-4846-ad1f-9eb2a83fd4ea)

压缩后：
![image](https://github.com/he3-app/he3-feedback/assets/25242380/3c776885-5c99-4189-b34c-7ca5f3eeda02)
![image](https://github.com/he3-app/he3-feedback/assets/25242380/4261832a-4176-4a27-b0fa-409272fbcb4a)

**Comment by lyzhang1999** [comment]
https://github.com/he3-app/he3-feedback/issues/245
